src/Microsoft.Azure.WebJobs.Host/DefaultParameterResolver.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// The default <see cref="ParameterResolver"/> used by the framework.
+    /// </summary>
+    public class DefaultParameterResolver : ParameterResolver
```

Comment:
Until this is truely pluggable by people at the JobHost level, I can just make this internal.